notes/PRODUCT.md

```diff
@@ -1,5 +1,7 @@
 # Product Thinking
-The art of defining and solving a problem with a specified target audience, strategy, and goal in mind.
+The art of defining and solving a problem with a specified target audience, strategy, and goal in mind.<br>
```

Comment:
in markdown, all you need to do is add a carriage-returned line between text sections, no need to add `<br/>` tags:

```
+ one line of text
+ 
+ second line of text
```

instead of 

```
+ one line of text <br/>
+ second line of text
```

Reply:
Thank you done.